fdbserver/storageserver.actor.cpp

```diff
@@ -5976,27 +5977,26 @@ ACTOR Future<Void> tryGetRangeFromBlob(PromiseStream<RangeResult> results,
                                        Reference<BlobConnectionProvider> blobConn) {
 	ASSERT(blobConn.isValid());
 	try {
-
 		state Standalone<VectorRef<BlobGranuleChunkRef>> chunks = wait(tryReadBlobGranules(tr, keys, fetchVersion));
-
 		if (chunks.size() == 0) {
 			throw blob_granule_transaction_too_old(); // no data on blob
 		}
-
 		if (!isRangeFullyCovered(keys, chunks)) {
-			throw blob_granule_transaction_too_old();
+			fmt::print("ERROR: keys not fully covered {}\n", keys.toString());
```

Comment:
What is the actual desired behavior when this happens?
If it is to swallow the issue and continue, we should log a trace event when this happens instead of a printf.

Reply:
fixed. we should throw the error.